codegen/src/main/java/software/amazon/awssdk/codegen/model/intermediate/MemberModel.java

```diff
@@ -429,6 +455,14 @@ public String getFluentSetterDocumentation() {
                + getEnumDoc();
     }
 
+    public String getExistenceCheckDocumentation() {
+        StringBuilder docBuilder = new StringBuilder();
+        docBuilder.append(StringUtils.isNotBlank(documentation) ? documentation : defaultExistenceCheck().replace("%s", name))
+                .append(LF);
+
+        return docBuilder.toString();
```

Comment:
Should we just use the default existence check text for all methods? I don't think it's particularly useful to have the documentation for the field on the has* method.

Current:
```Java
    /**
     * <p>
     * An attribute of type Binary Set. For example:
     * </p>
     * <p>
     * <code>"BS": ["U3Vubnk=", "UmFpbnk=", "U25vd3k="]</code>
     * </p>
     */
    public boolean hasBs() {
        return bs != null && !(bs instanceof SdkAutoConstructList);
    }
```

Reply:
🤦‍♂  Yes; will change.

Reply:
Fixed:
```
    /**
     * Returns true if the BS property was specified by the sender (it may be empty), or false if the sender did not
     * specify the value (it will be empty). For responses returned by the SDK, the sender is the AWS service.
     */
    public boolean hasBs() {
        return bs != null && !(bs instanceof SdkAutoConstructList);
    }
```